### PR TITLE
wip: Add logo to documentation

### PR DESCRIPTION
**Pull request by Mariatta**
Add logo file, and display the logo in documentation navigation menu

Closes https://github.com/coala/documentation/issues/216

Will this work? Let me know if anything needs adjustment. Thanks :)

screen shot 2016-10-24 at 9 28 46 pm

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -118,7 +118,9 @@
     'github_user': 'coala',
     'github_repo': 'coala',
     'github_banner': True,
-    'github_type': 'star'
+    'github_type': 'star',
+    'logo': '../../../coala_logo.png',
```
Comment (sils) [inline conf.py]: shouldn't this be put in some image directory? Then the path is probably also not that weird

**Comment by sils** [comment]
Maybe we should scale down the png to something like double the size it will be rendered in the end to improve pageload time? I think we need double for hidpi screens. Maybe @AbdealiJK has experience

**Comment by AbdealiJK** [comment]
Use the svg for the logo - smaller + self optimized for screen res/size
Link: https://github.com/coala/artwork/blob/master/artwork/logo/coala_color.svg

**Comment by sils** [comment]
can all browsers render that properly?

**Comment by AbdealiJK** [comment]
http://caniuse.com/#feat=svg -- More than 97% Only old android phones and old IE affected

**Comment by sils** [comment]
can browsers display that?

**Comment by Mariatta** [comment]
I think I messed up with git :( Some unrelated commits are showing up here. Any ideas of what I can do? Thanks

**Comment by sils** [comment]
Hi,

it looks like your PR is out of date and needs a rebase. [This page](http://coala.io/git) may help you to get started on this. (*Do not press the update branch button. It's evil.*) That should get rid of all the commits!

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala/documentation.git
```

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

Ping us on [our gitter channel](https://coala.io/chat) if you need help!

**Comment by sils** [comment]
or just ask on chat?

**Comment by sils** [comment]
if that doesn't work

**Comment by Mariatta** [comment]
Thanks :) I think it's all good now.
Please review the changes.
I'm using the svg file as suggested.

Here's the latest screenshot in this changeset.

screen shot 2016-10-30 at 3 29 25 pm

**Comment by AbdealiJK** [comment]
Ack 2e2ad2a048b5bc5682106768cd8a7ed3badc3702

**Comment by AbdealiJK** [comment]
@rultor merge

**Comment by sils** [comment]
oh @Mariatta that might have been a misunderstanding: I don't think we wanted the logo to be small, it should have the full width. The idea was to scale down the image to double the needed resoltion so browsers don't need to load a 2MB image just for the logo but more of a 20kb image - with the same width however.

**Comment by Mariatta** [comment]
@sils for sure, I can work on the logo later today. Do I commit it against this PR?

**Comment by sils** [comment]
No, we need a new pr

**Pull request merged**